multiarc/src/formats/rar/unrar/arccmt.cpp

```diff
@@ -1,3 +1,5 @@
+#include "rar.hpp"
```

Comment:
Please avoid any changes in rar/unrar directory as its bundled sources from Roshal and your changes will be overwritten by very next update on this sources. And they also will complicate update process.

Reply:
Better send that changes to Roshal directly :)